app/Http/Controllers/SessionController.php

```diff
@@ -78,13 +70,13 @@ public function init()
 					$return['status'] = Config::get('defines.status.LYCHEE_STATUS_LOGGEDIN');
 
 					$return['config'] = $this->configFunctions->public();
-					$return['lock'] = ($user->lock == '1');         // can user change their password
-					$return['upload'] = ($user->upload == '1');     // can user upload ?
+					$return['is_locked'] = $user->is_locked;    // can user change their password
```

Comment:
```suggestion
					$return['is_locked'] = $user->is_locked;    // can user change their password ?
```

Reply:
You cannot hide that you are French: you are blanking. Adding wrong spaces before punctuation marks in English. :stuck_out_tongue_closed_eyes: